testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -859,7 +859,7 @@ podman run --name postgres -p 5432:5432 -e POSTGRES_PASSWORD=pass -e POSTGRES_US
 To run the tests without spawning the container for you, execute tests with the following command:
 ```shell
 mvn clean install -f testsuite/integration-arquillian/tests/base \
-  -Pmap-storage,map-storage-jpa \
+  -Pmap-storage,map-storage-jpa-postgres \
   -Dpostgres.start-container=false \
   -Dkeycloak.map.storage.connectionsJpa.url=<jdbc_url> \
   -Dkeycloak.map.storage.connectionsJpa.user=<user> \
```

Comment:
Do you think it is worth to briefly mention `map-storage-jpa-cockroach` also as a valid profile that can be used to run the tests with CRDB?

Reply:
Updated in latest push.